app/controllers/weather_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WeatherController < ApplicationController
+
+  def forecast
+    @forecasts = WeatherService.new(params[:city]).get_forecast
+    render json: @forecasts
```

Comment:
What sort of issues could converting the forcast directly into JSON help to introduce in the future?